toolkit/scripts/filter-packages.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash -e
+
+function filter_packages {
+    # First argument is a path to the packages directory
+    rpms_directory="$1"
+
+    # Used later
+    fields_cutoff=2
+    if [[ $ignore_versions -eq 1 ]]; then
```

Comment:
>$ignore_versions [](http://example.com/codeflow?start=10&length=16)

Nit: this is a bit inconsistent. We pass the directories as an argument but use the global `ignore_versions`. I'd suggest to use only one approach. To make things cleaner, we could also make sure function-local variables are really local with the `local [variable]` declaration.